app/controllers/funding/trainee_summaries_controller.rb

```diff
@@ -31,11 +30,15 @@ def organisation
     end
 
     def trainee_summary
-      @trainee_summary ||= organisation&.funding_trainee_summaries&.order(:created_at)&.last
+      @trainee_summary ||= organisation&.funding_trainee_summaries&.where(academic_year: academic_year_string)&.order(:created_at)&.last
```

Comment:
I know it is existing code, but it is not good practice to use multiple safety operators `&` in a single query.